applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_small_sensitivities_primal_parameters.json

```
@@ -52,7 +52,7 @@
             "Parameters"    : {
                 "model_part_name" : "MainModelPart.Body2D_Body",
                 "velocity_infinity" : [10.0,0.0,0.0],
-                "create_output_file": false
+                "create_output_file": true
```

```suggestion
                "create_output_file": false
```
tests should not create output

---

you are using this in the test aren't you?
In this case please discard my comment

---

Ok, following your comment I found another way to check the results. You can find it here: https://github.com/KratosMultiphysics/Kratos/pull/4668/commits/d8d17e2995cdacab164df96b8bdcb7b6401ccc30